sdk/core/azure-core-tracing-opentelemetry/dev_requirements.txt

```diff
@@ -6,3 +6,4 @@ opentelemetry-instrumentation-requests>=0.32b0
 requests
 azure-storage-blob
 -e ../../servicebus/azure-servicebus
+-e ../../eventhub/azure-eventhub
```

Comment:
curious why you're doing an editable install here? I don't think this should be an issue, but if you ever also have the checkpointstoreblob package installed at the same time as the editable, it might give you an import/"" not found error. So if that happens, you'll prob have to make this a non-editable install.

Reply:
Yea, that's a good point, I had these editable because I had some debug statements in the eventhubs packages on the side while writing these. But no reason to keep these editable here!